### PR TITLE
Updates for 1.17.1 and 1.17.3

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: _includes/manuals/1.17/1.2_release_notes.md [hunk 1]
```diff
@@ -28,6 +28,54 @@ With auditing of host interfaces, it is now possible to see an IP address of a h
 ([#21374](http://projects.theforeman.org/issues/21374))
 
 
+### Release notes for 1.17.3 (1.17.2 was not released due to packaging issues)
```
Comment (ekohl) [inline _includes/manuals/1.17/1.2_release_notes.md]: I wouldn't say it was a packaging issue, it was a critical regression in VMware provisioning.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by xprazak2** [comment]
I changed the explanation why 1.17.2 was not released.

**Comment by ekohl** [comment]
I am wondering if Release notes 1.17.2 should be a separate section where the description is the body. I think that'll create a more usable table of contents. Other than that :+1:

**Comment by tbrisker** [comment]
Can you please include https://projects.theforeman.org/issues/24697 and https://projects.theforeman.org/issues/24640 in the upgrade warnings section?

**Review by tbrisker — APPROVED** [review]
Thanks @xprazak2

**Pull request merged**